package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rcs-core",
-  "version": "3.0.0-alpha.0",
```

Comment:
Please again do not update the version :)

Reply:
Yes, I know. This is because `rename-css-selectors` refuses to install with such "non semantic" version.

Reply:
Can you push the `next` tag to this repo (so at least `rename-css-selectors` can pass tests ?)

Reply:
I updated your PR (https://github.com/JPeer264/node-rename-css-selectors/pull/44). So the new `rcs-core` version is ready. Just run `yarn` before you begin to install the latest dependencies :)